examples/md-flexible/CMakeLists.txt

```diff
@@ -18,3 +18,15 @@ ADD_EXECUTABLE(md-flexible
 #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -qopt-report-phase=vec")
 TARGET_LINK_LIBRARIES(md-flexible
         autopas)
+
+#-----------------test-----------------
+# add check for current target
+add_test(
+        NAME md-flexible.test
```

Comment:
Fails with `Exception: Child aborted`. Fyi, this is not an error message you'll want to google.